src/site/site.xml

```diff
@@ -6,7 +6,7 @@
   <skin>
     <groupId>com.salesforce.maven</groupId>
     <artifactId>MavenSiteSkin</artifactId>
-    <version>1.0.16-SNAPSHOT</version>
+    <version>${project.version}</version>
```

Comment:
This, I believe will also be evaluated for the current maven execution, again breaking downstream projects.